Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.DataGridView/Krypton.Toolkit.Suite.Extended.DataGridView 2019.csproj

```diff
@@ -39,12 +52,7 @@
         <DocumentationFile>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.DataGridView\Krypton.Toolkit.Suite.Extended.DataGridView.xml</DocumentationFile>
         <OutputPath>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.DataGridView\</OutputPath>
     </PropertyGroup>
-  
-  <!--TODO: Help!!-->
-  <ItemGroup>
-    <!--<Reference Include="System.ValueTuple" />-->
-
-    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
-  </ItemGroup>
-  
+    <ItemGroup>
+      <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
I left some Options (4) on what you wanted to do to sort this out for .Net35 etc.

Reply:
Can it be solved by comparing the frameworks, i.e. if TFM >= netcoreapp3.1 then use <PackageReference Include="System.ValueTuple" Version="4.5.0" />, else use equivalent?

Reply:
Can it be solved by comparing the frameworks, i.e. if TFM >= netcoreapp3.1 then use <PackageReference Include="System.ValueTuple" Version="4.5.0" />, else use equivalent?

Reply:
It becomes messy very quickly.. I prefer option 4, and cut out everything that is not `T2`

Reply:
> It becomes messy very quickly.. I prefer option 4, and cut out everything that is not `T2`

Were the options in PR #216? If so, go for # 4 if that would resolve the issue :)

Reply:
No, I did the investigation before starting that.
Can you raise a separate bug / issue, and assign, and I'll do it tomorrow

Reply:
> No, I did the investigation before starting that.
> Can you raise a separate bug / issue, and assign, and I'll do it tomorrow

Done, I don't know where the comments went, but I've raised #217